libraries/liveIntentId/idSystem.js

```diff
@@ -43,6 +43,7 @@ const calls = {
 
 let eventFired = false;
 let liveConnect = null;
+let liModuleEnabled;
```

Comment:
Should this be `window.liModuleEnabled` ?

Reply:
This is actually what the linter was complaining about: https://github.com/LiveIntent/Prebid.js/pull/49#:~:text=Lint-,de9e9f0,-Merge%20info